Lib/test/test_generated_cases.py

```diff
@@ -479,4 +530,4 @@ def test_macro_cond_effect():
             DISPATCH();
         }
     """
-    run_cases_test(input, output)
+        self.run_cases_test(input, output)
```

Comment:
Would you mind adding
```py
if __name__ == "__main__":
    unittest.main()
```
to the end of the file? Sometimes it's handy to be able to run tests without invoking the full test harness.

Reply:
Done!